Userland/Utilities/image.cpp

```diff
@@ -176,6 +177,10 @@ static ErrorOr<void> save_image(LoadedImage& image, StringView out_path, bool pp
         TRY(Gfx::PortableFormatWriter::encode(*TRY(stream()), *frame, { .format = format }));
         return {};
     }
+    if (out_path.ends_with(".webp"sv, CaseSensitivity::CaseInsensitive)) {
```

Comment:
Guess what? It also needs to be added on line 193 :upside_down_face:

Reply:
I thought I'd save you the merge conflict… 😅

Done, thanks!

Reply:
CLion should be smart enough to figure out how to resolve the conflict